### ISSUE TITLE
error while running install script ./bootstrap.sh

### ISSUE THREAD

**Issue by joaodafonseca** [bakercp/ofxDlib#24]
Hi all,

while running the script i get the following error:

<img width="721" alt="screen shot 2018-05-18 at 17 24 25" src="https://user-images.githubusercontent.com/1932216/40243371-5a47b0f8-5ac0-11e8-9574-0df2e45bc542.png">

Any clue what could be the problem? 
Thanks in advance,
J

**Comment by bakercp** [comment]
what version of Xcode / command line tools / macOS are you using?

**Comment by joaodafonseca** [comment]
Xcode - 9.3.1
command line tools - not sure how to check this :(
macOS - 10.13.2

**Comment by joaodafonseca** [comment]
command Line Tools:  Xcode 9.3.1 (9E501)

**Comment by bakercp** [comment]
Hmm.  I think it is an Xcode 9.3 issue.  I haven't tested it with that (and dlib cmake scripts may not yet be compatible).  Can you try compiling dlib alone:

```
$ curl -LO http://dlib.net/files/dlib-19.11.tar.bz2
$ tar xvf dlib-19.11.tar.bz2
$ cd dlib-19.11
$ mkdir build
$ cd build/
$ cmake ..
```

If you get the same error as you were getting with the bootstrap script, you need to downgrade to Xcode 9.2 or raise the issue with the dlib repository, as it expects `AppleClang` as the compiler name, ideally submitting a PR to include the new `Clang` identifier in the cmake scripts.

**Comment by joaodafonseca** [comment]
Yes getting the same error. I will try to downgrade Xcode and will let you know

**Comment by bakercp** [comment]
OK.  Good to know.  Keep me posted.

**Comment by joaodafonseca** [comment]
Downgrading xCode to 9.2 doesn't really help, still having the same problem :(

**Comment by bakercp** [comment]
What does running `xcode-select --print-path` give you?

**Comment by joaodafonseca** [comment]
/Applications/Xcode.app/Contents/Developer

**Comment by bakercp** [comment]
I'd recommend posting an issue here:

https://github.com/davisking/dlib/issues

Post the results of running:

```
$ curl -LO http://dlib.net/files/dlib-19.11.tar.bz2
$ tar xvf dlib-19.11.tar.bz2
$ cd dlib-19.11
$ mkdir build
$ cd build/
$ cmake ..
```

**Comment by joaodafonseca** [comment]
ok will do thanks for help. I'll keep you posted..

**Comment by joaodafonseca** [comment]
@bakercp would be possible for you to send me a zip file with the files that are supposed to be inside the libs/dlib folder? I think the only problem I'm getting here is to install the dlib files.

**Comment by joaodafonseca** [comment]
So yeah I managed to run the examples by manually downloading flib and place the lib files in libs/dlib. Also found a compiled file "libdlib.a" that is needed in the ofxFacetracker2 addon. After that all examples run. It's not solution but a workaround for now.

**Comment by bakercp** [comment]
https://github.com/davisking/dlib/issues/1326

**Comment by vvzen** [comment]
I've got a different error during the _bootstrap.sh_ execution, but running
```
$ curl -LO http://dlib.net/files/dlib-19.11.tar.bz2
$ tar xvf dlib-19.11.tar.bz2
$ cd dlib-19.11
$ mkdir build
$ cd build/
$ cmake ..
```

works.
This is what I get after running _bootstrap.sh_:

![screen shot 2018-08-05 at 14 48 00](https://user-images.githubusercontent.com/10340139/43686473-958fc984-98be-11e8-920b-041b20cd07d9.png)

Any hints? (@bakercp , @joaodafonseca )

Thanks
